addons/website_sale_collect/tests/test_sale_order.py

```diff
@@ -35,6 +37,11 @@ def test_warehouse_is_not_reset_on_public_user_checkout(self):
         so.partner_id = self.partner.id
         self.assertEqual(so.warehouse_id, warehouse_2)
 
+    def test_warehouse_is_computed_based_on_pickup_location(self):
+        warehouse_2 = self._create_warehouse()
+        so = self._create_in_store_delivery_order(pickup_location_data=json.loads('{"id":' + str(warehouse_2.id) + '}'))
```

Comment:
We have a guideline to respect <=100 chars per line.
```suggestion
        so = self._create_in_store_delivery_order(
            pickup_location_data=json.loads('{"id":' + str(warehouse_2.id) + '}')
        )
```